plugin/mlsdbonlinelocator.cpp

```diff
@@ -45,8 +46,33 @@ MlsdbOnlineLocator::MlsdbOnlineLocator(QObject *parent)
     , m_modemManager(new QOfonoExtModemManager(this))
     , m_simManager(0)
     , m_networkManager(new NetworkManager(this))
+    , m_fallbacksLacf(true)
+    , m_fallbacksIpf(true)
     , m_currentReply(0)
 {
+    QFile gpsConf(QStringLiteral("/system/etc/gps_sfos.conf"));
```

Comment:
Does the file have to be in `/system`? It's a bit of a special place.

Reply:
What place should it be?

Reply:
in /etc/gps_sfos.ini

also it should be in qt ini format and is should be combined with gps_xtra.ini from geoclue-hybris

Reply:
One more reason to use QSettings to parse it

Reply:
https://bz.jollamobile.com/show_bug.cgi?id=41840#c1 I understand that the path must be /etc/gps_sfos.conf

Reply:
We shouldn't write anything to /system folder.

Reply:
I changed the folder.  "/etc/gps_sfos.ini"